README.md

```diff
@@ -1,3 +1,163 @@
+* [End user guide](#end-user-guide)
+    * [Get started](#get-started)
+    * [Use /drive commands](#use-drive-commands)
+    * [Post menu bindings](#post-menu-bindings)
+* [Development](#development-environment)
+    * [Manual installation](#manual-installation)
+    * [Install dependencies](#install-dependencies)
+    * [Run the local development environment](#run-the-local-development-environment)
+    * [Run the local development environment with Docker](#run-the-local-development-environment-with-docker)
+
+# Feature summary
+
+This application allows you to integrate Google Drive to your Mattermost instance, letting you:
+
+- Create a Google Drive file
+- Share a Google Drive file
+- View and reply to comments
+- Publish on Google Drive any file attached to a Mattermost post
+- Enable or disable notifications for all files
+
+# Set up
+
+## Installation HTTP
+
+To install, as a Mattermost system admin user, run the command ``/apps install http GOOGLE_API_URL`` in any channel. The ``/drive`` command should be available after the configuration has been successfully installed.
+
+The ``GOOGLE_API_URL`` should be replaced with the URL where the Google Drive API instance is running. Example: ``/apps install http https://myapp.com/manifest.json``
+
+## Installation Mattermost Cloud
+
+To install, as a Mattermost system admin user, run the command ``/apps install listed drive`` in any channel. The ``/drive`` command should be available after the configuration has been successfully installed.
+
+## Create a Google Cloud Project
+
+1. Create a new Project. You would need to redirect to [Google Cloud Console](https://console.cloud.google.com/home/dashboard) and select the option to **New project**. Then, select the name and the organization (optional).
+2. Select APIs. After creating a project, on the left side menu on **APIs & Services**, then, select the first option **Enabled APIs & Services** and wait, the page will redirect. 
+3. From the left menu select **Library** and activate: 
+    - Google Drive API
+    - Google Docs API
+    - Google Slides API
+    - Google Sheets API
+4. Go back to **APIs & Services** menu.
+5. Create a new OAuth consent screen. Select the option **OAuth consent screen**  on the menu bar. If you would like to limit your application to organization-only users, select **Internal**, otherwise, select **External** option, then, fill the form with the data you would use for your project.
+6. Go back to **APIs & Services** menu.
+7. Create a new Client. Select the option **Credentials**, and on the menu bar, select **Create credentials**, a dropdown menu will be displayed, then, select **OAuth Client ID** option. 
+8. Then, a select input will ask the type of Application type that will be used, select **Web application**, then, fill the form, and on **Authorized redirect URIs** introduce this URI:
+    https://<your_mattermost_instance>/plugins/com.mattermost.apps/apps/Drive/oauth2/remote/complete
```

Comment:
I am getting the redirect_url mismatch error with this URL endpoint. 
`https://<your_mattermost_instance>/plugins/com.mattermost.apps/apps/Drive/oauth2/remote/complete`

The expected URL looks like:
`https://<your_mattermost_instance>/plugins/com.mattermost.apps/apps/google-drive/oauth2/remote/complete`

Reply:
This is resolved in commit [6d9ef5d](https://github.com/mattermost/mattermost-app-google-drive/pull/2/commits/6d9ef5dd391b63393237c569a1efad004dd19c7d)